Tasks/PublishSymbols/PublishSymbols.ps1

```diff
@@ -68,13 +68,22 @@ try {
 
     if ( ($SymbolServerType -eq "FileShare") -or ($SymbolServerType -eq "TeamServices") -or (-not $SkipIndexing) ) {
         # Get the PDB file paths.
-        [string]$SearchPattern = Get-VstsInput -Name 'SearchPattern' -Default "**\bin\**\*.pdb"
-        # $pdbFiles = @(Find-VstsFiles -LiteralDirectory $SymbolsFolder -LegacyPattern $SearchPattern)
-        $pdbFiles = @(Find-VstsMatch -DefaultRoot $SymbolsFolder -Pattern $SearchPattern)
+        [string]$SearchPattern = Get-VstsInput -Name 'SearchPattern' -Default "**\*.pdb"
```

Comment:
>**\*.pdb [](start = 79, length = 8)

Keep as "**\bin\**\*.pdb" as this is a breaking change.

You are right though :)